src/main/java/com/vacation_bot/core/customization/CustomizationService.java

```diff
@@ -98,6 +98,8 @@ private String findAndReplacePersonNames( String sentence, CustomizedSentence cu
                 }
             }
         }
+        //TODO: what to do with duplicated names? For right now lets imagine that names are unique. It looks like we should find a way to set here the userId
+        // It should be possible to retrieve userId directly from Slack
```

Comment:
Ideal case for duplicates is to resolve them by asking additional question. E.g. seeing name John, bot asks: Smith or Doe? I like idea with Slack user ids. Yet for admins it would be important to see other users vacation status.